### PR TITLE
fix: make border radius challenge Firefox compatible

### LINKED ISSUE #34867: [FIREFOX] Unable to pass Basic CSS: Add Rounded Corners with border-radius
**Describe your problem and how to reproduce it:**

Unable to pass challenge in fiefox, chrome is OK

**Add a Link to the page with the problem:**

http://localhost:8000/learn/responsive-web-design/basic-css/add-rounded-corners-with-border-radius

**Tell us about your browser and operating system:**

* Browser Name: FireFox
* Browser Version: 64.0.2 (64-bit)
* Operating System: Mac 10.14.2

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

image

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #34867 by changing the tests to check the radii of all four corners separately.

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-css/add-rounded-corners-with-border-radius.english.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ tests:
     <button type="submit">Submit</button>
   </form>
 </main>
+<script>
```
Comment (Bouncey) [inline curriculum/challenges/english/01-responsive-web-design/basic-css/add-rounded-corners-with-border-radius.english.md]: @ojeytonwilliams I think we should remove this

**Review by Bouncey — CHANGES_REQUESTED** [review]

**Comment by ojeytonwilliams** [comment]
Yeah, sorry about that!

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
Thanks @ojeytonwilliams

**Comment by RandellDawson** [comment]
@Bouncey I assume the same change needs to be made across all the other languages?  If so, @ojeytonwilliams would you be interested in updating those as well?

**Comment by ojeytonwilliams** [comment]
Sure, I'd be happy to take a look.

**Comment by ojeytonwilliams** [comment]
@Bouncey One thing I'm unsure about is the following: the translated tests typically fail at the validation stage (missing descriptions, syntax errors and so on).  Would it be better if I tried to make it pass the validation or just change the tests.

I'm a little unsure, because some of the fixes would require me to add placeholder data just to validate correctly.

**Pull request merged**